examples/typescript/preprocessor.js

```diff
@@ -0,0 +1,16 @@
+var tsc = require('typescript');
+
+module.exports = {
+  process: function(src, path) {
+    // TypeScript files can be .ts, .tsx
+    var result = src;
+    if(path.slice(-3) === '.ts' || path.slice(-4) === '.tsx'){
```

Comment:
I’ve chosen "slice", because "endsWith" requires ES6.
Thinking 1.6 of TypeScript requires Node.js has already implemented "endsWith".
And also "endsWith" comparing to "slice", more straightforward because less parameters.
So I think your request appropriate.
I will fix and test soon.